swf/src/bin/prepare_playerglobal.rs

```diff
@@ -0,0 +1,40 @@
+//! An internal Ruffle utility to perform post-processing
```

Comment:
Let's move this to `core/src/avm2/playerglobal/`, or even better, integrate it into `core/build.rs` (if it's makes sense).

Reply:
I'd like `compile_swc.sh` to remain usable as a standalone script, which means that `prepare_playerglobal` needs to be a binary. Should I move it to `core/src/bin/` instead?